airbyte-integrations/connectors/source-smartsheets/source_smartsheets/spec.json

```diff
@@ -34,8 +38,8 @@
   },
   "advanced_auth": {
     "auth_flow_type": "oauth2.0",
-    "predicate_key": [],
-    "predicate_value": "",
+    "predicate_key": ["auth_method"],
+    "predicate_value": "oauth2.0",
```

Comment:
I believe the FE team was working on making it so that an empty predicate key means to always show the button, so if this is the intention then this shouldn't be necessary.

Slack thread: https://airbytehq-team.slack.com/archives/C02UF50V9HA/p1665764625977619